pipelines/ApVerifyWithFakes.yaml

```diff
@@ -3,211 +3,51 @@
 
 description: Fully instrumented AP pipeline with fakes
 imports:
-  - location: $AP_VERIFY_DIR/pipelines/ApVerify.yaml
-    exclude:
-      - fracDiaSourcesToSciSources  # Selectively turn off this task's contracts
+  - location: $AP_PIPE_DIR/pipelines/ApPipeWithFakes.yaml
+  - location: $AP_VERIFY_DIR/pipelines/MetricsRuntime.yaml
+  - location: $AP_VERIFY_DIR/pipelines/MetricsMisc.yaml
   - location: $AP_VERIFY_DIR/pipelines/MetricsForFakes.yaml
-parameters:
-  fakesType: fakes_
-  # TODO: redundant connection definitions workaround for DM-30210
-  fakesInput: fakes_fakeSourceCat
-  fakesPvi: fakes_calexp
-  fakesTemplate: fakes_goodSeeingCoadd
-  fakesDcrTemplate: fakes_dcrCoadd
-  fakesDiaSrcCat: fakes_goodSeeingDiff_diaSrc
-  fakesDiaSrcSchema: fakes_goodSeeingDiff_diaSrc_schema
-  fakesDiaSrcParquet: fakes_goodSeeingDiff_diaSrcTable
-  fakesDiff: fakes_goodSeeingDiff_differenceExp
-  fakesDiffScore: fakes_goodSeeingDiff_scoreExp
-  fakesDiffWarp: fakes_goodSeeingDiff_warpedExp
-  fakesDiffMatch: fakes_goodSeeingDiff_matchedExp
-  fakesAssocSrc: fakes_goodSeeingDiff_assocDiaSrc
-  fakesMatchedSrc: fakes_goodSeeingDiff_matchDiaSrc
-  # TODO: end DM-30210 workaround
 tasks:
-  createFakes:
-    class: lsst.ap.pipe.createApFakes.CreateRandomApFakesTask
-    config:
-      magMin: 20
-      magMax: 26
-      fakeDensity: 2000
-      connections.fakesType: parameters.fakesType
-      # TODO: redundant connection definitions workaround for DM-30210
-      connections.fakeCat: parameters.fakesInput
-      # TODO: end DM-30210 workaround
-  coaddFakes:
-    class: lsst.pipe.tasks.insertFakes.InsertFakesTask
-    config:
-      doSubSelectSources: True
-      select_col: "isTemplateSource"
-      connections.coaddName: parameters.coaddName
-      connections.fakesType: parameters.fakesType
-      # TODO: redundant connection definitions workaround for DM-30210
-      connections.image: parameters.template
-      connections.fakeCat: parameters.fakesInput
-      connections.imageWithFakes: parameters.fakesTemplate
-      # TODO: end DM-30210 workaround
-  singleFrameWithFakes:
-    class: lsst.pipe.tasks.processCcdWithFakes.ProcessCcdWithVariableFakesTask
-    config:
-      insertFakes.doSubSelectSources: True
-      insertFakes.select_col: "isVisitSource"
-  retrieveTemplateWithFakes:
+  # Parallel to retrieveTemplateWithFakes, allows non-fakes image differencing.
```

Comment:
Here and below, please clarify in the comment that you're referring to how ApPipe labels the task (it took me a minute to figure out what it was parallel to)